OpenRA.Mods.Common/Projectiles/LaserZap.cs

```diff
@@ -128,15 +156,15 @@ public void Tick(World world)
 				target = blockedPos;
 			}
 
-			if (!doneDamage)
+			if (ticks < info.DamageDuration && --interval <= 0)
 			{
 				if (hitanim != null)
```

Comment:
Do we really want the hitanim block into this if? I'd presume this block should be only ran in the first tick and not in all damaging ticks OR we want new hitanims played in all damaging ticks. This doesn't cover either.